kernel/ukvm/kernel.c

```diff
@@ -29,13 +29,16 @@ void _start(void *arg)
     platform_init(arg);
     cmdline = cmdline_parse(platform_cmdline());
 
+    console_init();
```

Comment:
Yes this would work since the Muen console driver has no initialization dependencies on other parts of the system. Will move the call above **cpu_init()**.